client/src/components/composables/useConfig.js

```diff
@@ -0,0 +1,15 @@
+import { computed, onMounted } from "vue";
+import store from "store";
+
+/* composable config wrapper */
+export function useConfig() {
+    const config = computed(() => store.getters["config/config"]);
+    const isLoaded = computed(() => store.getters["config/configIsLoaded"]);
+
+    // Anytime we mount this (for now), make sure to load.
+    onMounted(() => {
+        store.dispatch("config/loadConfigs");
+    });
+
+    return { config, isLoaded };
+}
```

Comment:
Since unlike providers, composables are not components, do we want to keep them in a components sub-directory, or maybe move them somewhere else? In  #14464 I've added them to a separate directory, but I can move them to components if that approach is preferable.

Reply:
@ElectronicBlueberry Good catch, yep, it shouldn't live in components anymore.